sdks/python/gen_protos.py

```diff
@@ -15,40 +15,100 @@
 # limitations under the License.
 #
 
-"""Generates Python proto modules and grpc stubs for Beam protos."""
+"""
+Generates Python proto modules and grpc stubs for Beam protos.
+"""
+
 import contextlib
 import glob
 import inspect
 import logging
-import multiprocessing
 import os
 import platform
 import re
 import shutil
 import subprocess
 import sys
 import time
-import warnings
+from collections import defaultdict
 from importlib import import_module
 
 import pkg_resources
 
+LOG = logging.getLogger()
+LOG.setLevel(logging.INFO)
+
+LICENSE_HEADER = """
```

Comment:
I would just drop these two headers and skip the checks over adding logic to prepend them to the auto-generated files.

Reply:
If you're not strongly opposed, I suggest we keep these. My argument is less related to the build system, and more related to messaging to consumers of the SDK: if they are exploring the SDK, it is a useful notice to have so that they know this is Beam's posture